app/controllers/api/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::SessionsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def show 
+    current_user ? head(:ok) : head(:unauthorized)
+  end
+
+  def create 
+    user = User.where(email: params[:email]).first
+
+    if user&.valid_password?(params[:password])
+      render json: :create, status: :created
+    else
+      head(:unauthorized)
+    end
+  end
+
+  def destroy
+    current_user&.authentication_token = nil
+    if current_user.save
+      head(:ok)
+    else
+      head(:unauthorized)
```

Comment:
This error message is wrong, the problem is not the user is unauthorized, please read an follow this useful guide for error handling, your front end will thank you 
https://nordicapis.com/best-practices-api-error-handling/